### PR TITLE
Update license

### PR DESCRIPTION
**Pull request by paul-sachs**
Add shared tool to keep license up to date.

### CODE REVIEW

File: packages/protoc-gen-connect-query-react/package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
   "scripts": {
     "clean": "rm -rf dist",
     "test": "jest",
-    "build": "pnpm clean && tsc --project tsconfig.build.json"
+    "build": "pnpm clean && tsc --project tsconfig.build.json",
+    "license-header": "license-header --ignore 'dist/**' --ignore 'coverage/**'"
```
Comment (timostamm) [inline packages/protoc-gen-connect-query-react/package.json:25]:
```suggestion
    "license-header": "license-header"
```

See https://github.com/bufbuild/jest-environment-jsdom/pull/3

File: packages/protoc-gen-connect-query/package.json [hunk 2]
```diff
@@ -21,7 +21,8 @@
   "scripts": {
     "clean": "rm -rf dist",
     "test": "jest",
-    "build": "pnpm clean && tsc --project tsconfig.build.json"
+    "build": "pnpm clean && tsc --project tsconfig.build.json",
+    "license-header": "license-header --ignore 'dist/**' --ignore 'coverage/**'"
```
Comment (timostamm) [inline packages/protoc-gen-connect-query/package.json:25]:
```suggestion
    "license-header": "license-header"
```

**Review by timostamm — COMMENTED** [review]
I think you'll need to update the copyright years in the test expectations 😝

File: packages/protoc-gen-connect-query/package.json (hunk 2 above)
Comment (paul-sachs) [reply packages/protoc-gen-connect-query/package.json]: I instead moved the license-header call to root, that way we don't have a bunch of little scripts doing the exact same thing.

**Review by timostamm — APPROVED** [review]

**Comment by paul-sachs** [comment]
Closes #9

**Pull request merged**